backend/models/domain/order.js

```diff
@@ -7,15 +7,36 @@ const orderSchema = mongoose.Schema({
 		ref: 'User',
 	},
 	orderItems: [{
-		name: {
+		title: {
+			type: String,
+			required: true,
+		},
+		cover_image: {
 			type: String,
 			required: true,
 		},
 		price: {
-			type: Number,
-			required: true
+			amount: {
+				type: Number,
+				required: true,
+			},
+			currency: {
+				type: String,
+				required: true,
+			},
+			currency_symbol: {
+				type: String,
+				required: true,
+			},
+			price_string: {
+				type: String,
+			},
+			is_free: {
+				type: Boolean,
+				default: false,
+			},
 		},
-		course: {
+		courseId: {
```

Comment:
Creo que no debería tener una relación con curso, un curso puede cambiar, pero los datos que tiene en la  orden no se cambia. Creo que solo con los datos que ya tiene del price, title and cover ya esta bien y mantener el ID del curso como string. Mantener en Mongo la referencia solo en pocos casos

Reply:
De acuerdo! Tuve problemas a la hora de relacionar el curso y la orden y por la hora no encontré otra solución. Ya lo cambio!

Reply:
ok np, tengo apuntado de mirar toda la dependencia entre los documentos.

Reply:
@leonardopache Listo ya quedo!